enterprise-java.html.slim

```diff
@@ -0,0 +1,67 @@
+
+.wide.microsite-text-points
+  .row
+    .large-6.column
+      h4 Learn About Community
+      asciidoc:  
+		 Work alongside smart, motivated people to apply your skills toward a bigger effort. Look at the link:http://javaee-samples.github.io/[Java EE 7 Samples Project] or contribute to link:https://github.com/jboss-developer/www.jboss.org[JBoss Developer] and don't forget to find solutions in our link:http://www.jboss.org/forums/[Forums].
+    .large-12.column
+      h4 Getting Started Is Easy
+      asciidoc:   
+		Find all you need in the link:http://www.jboss.org/products/eap/get-started/[Getting Started Section] on JBoss developer. It's not much more than just hitting download and unzip into a folder of your choice. And we also provide link:http://www.jboss.org/products/eap/developer-materials[Developer Materials] and link:http://www.redhat.com/en/services/training[Training] to improve your skills, or the skills of your team, with our hands-on, role-based training for IT professionals and developers. If you feel the need for latest innovation and specifications, you need to look at link:http://wildfly.org/[WildFly]. This is the community version and helps you build flexible and lightweight applications.
+    .large-6.column
+	h4 Learn About Technology	
+	asciidoc:     
+		There are many resources available for link:http//:www.jboss.org/products/eap/overview/[Red Hat JBoss Enterprise Application Platform] on link:http://www.jboss.org[JBoss Developer] or the link:http://access.redhat.com[Red Hat Customer Portal]. You can also follow our Developer Advocates link:http://www.arungupta.me[Arun Gupta] or link://blog.eisele.net[Markus Eisele] and see, what they have to share.
+
+.wide.microsite-large-icons
+  .row
+    .large-4.column.center.microsite-large-icon
+      i.fa.fa-map-marker
+    .large-20.column
+      h3 Supported Middleware vs. Upstream Projects
+      p Our supported product portfolio provides enterprise-ready software built from an innovative community of projects. It delivers enterprise-class stability, enterprise-class support SLAs, guaranteed patches, updates and hot-fixes, and legal assurance. We achieve this by challenging convention and redefining Enterprise Java through community-driven projects. Community projects represent the latest technologies for use in cutting-edge applications and offer best-effort, community support.
+	a.button(href="http://www.jboss.org/technology/") Learn More
+
+.row	
+	.large-8.columns
+		= video_player(site.videos["https://www.youtube.com/v=xa_gtRDpwyQ"])	
+	.large-8.columns
+		= video_player(site.videos["https://www.youtube.com/v=kEYVBtkjAE4"])	
+	.large-8.columns
+		= video_player(site.videos["https://www.youtube.com/v=cMnPrEiYez0"])	
```

Comment:
site.videos is a collection of videos that are indexed by the site. They are keyed on the url. For this to work, the URL needs to represent a video already indexed. You can follow these instructions: https://github.com/jboss-developer/www.jboss.org/blob/master/CONTRIBUTING.md#to-add-a-youtube-video-to-the-site

Then you either need to wait for the cache to clear. This takes upto 6hrs, or I can force it for you.